builder/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "builder"
+build = "build/main.rs"
+rust-version.workspace = true
+edition.workspace = true
+version.workspace = true
+license.workspace = true
+
+[features]
+default = ["profiling"]
```

Comment:
Not really. Since telemetry and data-pipeline symbols are reexported through profiling I assumed that prof was kind of the default feature. Should we leave it out?

Reply:
This `default` feature for builder is the thing that makes the benchmark times change. Don't ask me to explain how.

Reply:
@morrisonlevi any more thoughts on this?

Reply:
As a contributing member to the profiling subtree, I am happy to not ship it as default. We don't need special treatment, as far as I am aware.